axelrod/tests/unit/test_tournament.py

```diff
@@ -177,44 +304,74 @@ def test_progress_bar_play(self):
             turns=axelrod.DEFAULT_TURNS,
             repetitions=self.test_repetitions)
 
-        results = tournament.play()
+        err = io.StringIO()
+        with redirect_stderr(err):
+            results = tournament.play()
         self.assertIsInstance(results, axelrod.ResultSet)
-        self.assertEqual(tournament.progress_bar.total, 15)
```

Comment:
I'd like these to stay in. This complemented by your file reading give a comprehensive test of the progress bar (without it, the total could be wrong for example).